### ISSUE TITLE
Accessibility rollup for dynamic code

### ISSUE THREAD

**Issue by sfisher** [CDL-Dryad/dryad-product-roadmap#603]
**Note: do not address upload page accessibility changes as this page will be re-done in the new year**

These are some accessibility problems Joel found but may be hard for him to address without getting into the rails or javascript code slightly and so one of the developers needs to look at.

- [x] On the File/URL uploading page for submitting a dataset:
  - [x] There is a DIV that has the text "upload complete." Change this to be a general message area for all messages having to do withe file uploads.
  - [x] When a file is deleted, have a message display in the div mentioned above as well as making it disappear from the table.
  - [x] When a file-upload session completes, have it display here.
  - [x] ~~In URL upload submissions, if there are validation events, have it say something in there (URLs validated, validation problems, etc).~~
- [x] ~~In the metadata entry page for relationships to the dataset ( `dryad-app/stash/stash_datacite/app/views/stash_datacite/related_identifiers/_form.html.erb`): Make the labels for match the form field ID that is dynamically generated.~~  Changed the way this works and no longer applicable.
- [x] Geoblacklight explore page--if possible but may be difficult since this is 3rd party library:
  - [x] Change "Find by location" to "Find by location in map" so it's more clear what is going on to non-sighted.
  - [x] Can indicate that map can be skipped to screen reader since there will never be labels for every image tile in a visual map.
- [x] Search results page from geoblacklight--if reasonable to customize the page layout without a lot of problems:
  - [x] Add an H1 somewhere that says "Search Results"
- [x] On the dataset landing page if you're a dataset owner:
  - [x] add a field label for the "select text" box that is for a sharing URL
  - [x] Link to ORCID icon with green blob, change to a span instead of a link with nothing inside.  This same icon may occur a few other places.  Can't have link without text.
- [x] Joel suggests using buttons, not links for changing the forms on the page for things like "add another author."  I suppose this makes some sense but those items do not cause submission of data or data changes by themselves so the standard for this isn't really clear cut and he mentioned this isn't really an accessibility problem (ie, optional).
- [x] The keywords entry field has a problem when trying to TAB out of and doesn't go to the next field while entering one.
- [x] The file upload page uses radio buttons to change pages for quite different workflow and functionality that look kind of the same.  It would be better to change these to a link to do the other kind of data entry rather than a radio button.
